src/overlays/actors/ovl_Bg_Ice_Shelter/z_bg_ice_shelter.c

```diff
@@ -114,7 +131,10 @@ void func_80890874(BgIceShelter* this, PlayState* play, CollisionHeader* collisi
     }
 }
 
-void func_808908FC(Vec3f* dest, Vec3f* src, s16 angle) {
+/**
+ * Used to convert the position offset for the steam line particles from relative to the ice wall to absolute.
+ */
+void BgIceShelter_ConvertToAbsolutePos(Vec3f* dest, Vec3f* src, s16 angle) {
```

Comment:
another copypasted (?) function, but it's so simple it may also have been rewritten every time idk

This one hasn't been named consistently but I know it can be found in a bunch of actors, for example `ObjSwitch_RotateY`

I think it's better to name it after the trigonometry operation it does than the use it gets

The comment could go near the call to the function instead of describing the function itself?

Reply:
I renamed it to `BgIceShelter_RotateY` and moved the comment.